src/base/detoken.lua

```diff
@@ -49,6 +49,7 @@
 			-- convert the token into a function to execute
 			local func, err = loadstring("return " .. token)
 			if not func then
+				print("load error:", err)
```

Comment:
We shouldn't need to print the error here though right? Because it falls through to an `error()` in the caller?